--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -246,7 +246,7 @@
 /* These u-blox products use Qualcomm's vendor ID */
 #define UBLOX_PRODUCT_R410M			0x90b2
 /* These Yuga products use Qualcomm's vendor ID */
-#define YUGA_PRODUCT_CLM920_NC5			0x9625
+#define YUGA_PRODUCT_CLM920_NC5			0x9025
 
 #define QUECTEL_VENDOR_ID			0x2c7c
 /* These Quectel products use Quectel's vendor ID */
